pkg/services/object/put/distributed.go

```diff
@@ -155,6 +155,9 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 
 	err := t.saveObject(*t.obj, objMeta, t.encodedObject)
 	if err != nil {
+		if errors.Is(err, apistatus.ErrIncomplete) {
+			return t.obj.GetID(), nil
```

Comment:
so we do not return an incomplete error in that case? or i got smth wrong?

Reply:
Good catch, fixed.